src/components/space/ExportData.js

```diff
@@ -0,0 +1,148 @@
+import React, { useContext, useState } from 'react';
+import { Tooltip } from '@material-ui/core';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import LaunchIcon from '@material-ui/icons/Launch';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import CloudDownloadIcon from '@material-ui/icons/CloudDownload';
+import { TaskDataContext } from '../../contexts/TaskDataProvider';
+
+const LightTooltip = withStyles((theme) => ({
```

Comment:
Maybe refactor to its own component under `common`? Eventually this could be imported from Graasp UI! :-)